go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/ElrondNetwork/elrond-go-core v1.1.16-0.20220523150518-f1519c41d352
 	github.com/ElrondNetwork/elrond-go-crypto v1.0.1
 	github.com/ElrondNetwork/elrond-go-logger v1.0.7
-	github.com/ElrondNetwork/elrond-vm-common v1.3.7-0.20220602122113-1f938c4d517a
+	github.com/ElrondNetwork/elrond-vm-common v1.3.7
```

Comment:
are we sure this is the correct commit?

Reply:
Yes. I did a proper release after PR https://github.com/ElrondNetwork/elrond-vm-common/pull/92 was merged in the `master` branch